pools/usecase/pools_usecase.go

```diff
@@ -316,7 +321,7 @@ func (p *poolsUseCase) GetPools(opts ...domain.PoolsOption) ([]sqsdomain.PoolI,
 			}
 
 			// Check filter is non-zero to avoid more expensive get liquidity cap check.
-			if pool.GetLiquidityCap().Uint64() >= options.MinPoolLiquidityCap {
+			if options.MinPoolLiquidityCap == 0 || pool.GetLiquidityCap().Uint64() >= options.MinPoolLiquidityCap {
```

Comment:
Seems that Paddy's bug stemmed from this while testing v26.x

I could not repro the panic on v25.x but I added the safeguard just in case.